### PR TITLE
adding openmm-ml again

### PR DESCRIPTION
**Pull request by mikemhenry**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mikemhenry** [comment]
I am willing to maintain this package

**Comment by mikemhenry** [comment]
@conda-forge/staged-recipes ready for review! We are in the middle of coordinating the simultaneous release of a few packages so I would appreciate a temporary relaxation of the policy to use `git_url` Once we get it in a staged recipe and get everything lined up, we will switch to using tarballs for the release.

**Comment by mikemhenry** [comment]
Friendly bump, ready for review :)
@conda-forge/help-python

**Comment by mikemhenry** [comment]
@jaimergp when you get a chance, can you review this package?  See this comment about the `git_url` https://github.com/conda-forge/staged-recipes/pull/19223#issuecomment-1194294941

**Comment by mikemhenry** [comment]
@conda-forge/staged-recipes ready for review!

File: recipes/openmm-ml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy
+    - openmm >=7.7 # TODO https://conda-forge.org/docs/maintainer/knowledge_base.html#creating-a-pre-release-build once merged into a feedstock, depend on dev for now
```
Comment (carterbox) [inline recipes/openmm-ml/meta.yaml:23]: > We are in the middle of coordinating the simultaneous release of a few packages so I would appreciate a temporary relaxation of the policy to use git_url Once we get it in a staged recipe and get everything lined up, we will switch to using tarballs for the release.

I think if you add the `conda_build_config.yaml` with channel targets to this PR, this package will be published to your pre-release label. Is there a reason you can't do all the coordination under a pre-release label?

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/openmm-ml/meta.yaml (hunk 1 above)
Comment (mikemhenry) [reply recipes/openmm-ml/meta.yaml:23]: @carterbox we can! I wasn't aware I could use that option before the staged recipe was migrated to a feedstock.

**Comment by mikemhenry** [comment]
@conda-forge/staged-recipes ready for re-review!

**Review by carterbox — APPROVED** [review]
I think this will work. The only issue I have with this recipe is downloading via commit tag instead of tarball, but it should be published to the rc channel instead of the main one.

**Pull request merged**